ha/util/conf_store.py

```diff
@@ -64,7 +64,7 @@ def get_server_pods(index):
         Get machine ids for server pods: returns list of machine ids.
         """
         machine_ids = []
-        server_pod_keys = Conf.search(index, GconfKeys.NODE_CONST.value, GconfKeys.SERVICE_CONST.value, Const.SERVICE_S3_HAPROXY.value)
+        server_pod_keys = Conf.search(index, GconfKeys.NODE_CONST.value, GconfKeys.NAME_CONST.value, Const.COMPONENT_RGW.value)
```

Comment:
Is this expected to be backward compatible with S3 server ? @shaileshsvc  any inputs ?

Reply:
Shailesh confirmed this in the DSU today. That is why I have used and started the PR.